alarm/utils.py

```diff
@@ -33,3 +32,65 @@ def any_of_trade_pair_thresholds_is_broken(trade_pair):
             return True
 
     return False
+
+
+def create_thresholds_brakes_from_recent_candles_update(trade_pair):
+    thresholds = Threshold.objects.filter(trade_pair=trade_pair)
+    for threshold in thresholds:
+        ThresholdBrake.objects.create(threshold=threshold)
+
+
+def get_thresholds_brake_prices(trade_pair):
+    three_second_ago = timezone.now() - timezone.timedelta(seconds=3)
+    thresholds_brake_prices = ThresholdBrake.objects.filter(threshold__trade_pair=trade_pair,
+                                                            happened_at__gte=three_second_ago).values_list(
+        'threshold__price', flat=True)
+    return thresholds_brake_prices
+
+
+def get_trade_pair_current_price(trade_pair):
+    last_candle = Candle.last_for_trade_pair(trade_pair)
+    if last_candle:
+        return last_candle.high_price
+    else:
+        return None
+
+
+def create_message_for_threshold_break(trade_pair):
+    trade_pair_str = format_trade_pair_for_message(trade_pair)
+
+    thresholds_brake_prices = get_thresholds_brake_prices(trade_pair)
+
+    thresholds_brake_prices_str = ', '.join([f'{price}$' for price in thresholds_brake_prices])
+
+    trade_pair_current_price = get_trade_pair_current_price(trade_pair)
+
+    message = f"Attention! Trade pair {trade_pair_str} has broken thresholds {thresholds_brake_prices_str} "
+    message += f"and current price is {trade_pair_current_price}$"
```

Comment:
Why do we split one message creation on two operations?

Reply:
I changed the message to one line